scripts/update-documentation-md.ts

```diff
@@ -0,0 +1,25 @@
+import { resolve } from 'node:path';
+import { unixEOL } from '@sap-cloud-sdk/util';
+import { transformFile } from './util';
+import { exit } from 'node:process';
+
+async function updateDocumentationMd() {
+  if (!process.env.NEXT_PACKAGE_VERSION) {
+    throw new Error('NEXT_PACKAGE_VERSION is not set!');
+  }
+  await transformFile(resolve('DOCUMENTATION.md'), documentation =>
+    documentation
+      .split(unixEOL)
+      .map(line =>
```

Comment:
Can't you use a simple .replace() instead of .split.map.join?

Reply:
Probably, I am not 100% what other implications this might have -> apparently there might be multiple lines that look like this. I don't dare touching it really, because there are no tests for it.